README.md

```diff
@@ -0,0 +1,24 @@
+	![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+This is my Microverse project called "Portfolio", it has a (toolbar or header and a headline section).
+
+![image](https://user-images.githubusercontent.com/74814780/129695103-8779e9f2-0d70-401d-a5fb-ebfcdf5b9445.png)
+
+
+Below are some features of this project:
+
+1. Has a header or toolbar section.
+
+2.  Has a headline section introducing me.
+
+Built with :
+
+HTML and CSS
+
+Linter styling to help debug and make the code neater.
+
+Acknowledgement:
+
+The idea was gotten from the Microverse Figma template.
```

Comment:
- [ ] Let us improve our Read-me file to provide the necessary information regarding our project. Kindly use this [template](https://github.com/microverseinc/readme-template) for more details. All in all README file should be descriptive.